CMakeLists.txt

```
@@ -99,6 +99,27 @@ if(JACK)
   install(FILES cmake/modules/FindJACK.cmake DESTINATION "${CMAKE_INSTALL_LIBDIR}/cmake/portaudio/modules")
 endif()
 
+# PulseAudio is mostly for Linux
+
+find_package(PULSEAUDIO)
+cmake_dependent_option(PULSEAUDIO "Enable support for PulseAudio general purpose sound server" ON PULSEAUDIO_FOUND OFF)
+if(PULSEAUDIO)
+  target_link_libraries(PortAudio PRIVATE ${PULSEAUDIO_LIBRARIES})
```

```suggestion
  target_link_libraries(PortAudio PRIVATE PulseAudio::PulseAudio)
```
export an INTERFACE library target from the module